lib/executor/engine/engine.cpp

```diff
@@ -47,29 +47,19 @@ Executor::runFunction(Runtime::StackManager &StackMgr,
   }
 
   // Enter and execute function.
-  AST::InstrView::iterator StartIt = {};
-  Expect<void> Res = {};
-  if (auto GetIt = enterFunction(StackMgr, Func, Func.getInstrs().end())) {
-    StartIt = *GetIt;
-  } else {
-    if (GetIt.error() == ErrCode::Value::Terminated) {
```

Comment:
I just want to know if we don't need this `Unexpected Return` after the refactoring.